packages/composites/bob/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "@chainlink/bob",
+  "version": "0.0.2",
+  "description": "",
+  "author": "Evangelos Barakos (evangelos@smartcontract.com) Silas Lenihan (silas@smartcontract.com)",
```

```suggestion
  "author": "Silas Lenihan (silas@smartcontract.com)",
```

Credit where credit is due, even if from a template this was all you :wink: